### PR TITLE
Updates 16.1 quests to official scripts

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: -

* **Server Mode**: renewal

* **Description of Pull Request**: 

Content
---------------------------------------------------
This PR converts the official 16.1 scripts, replacing the custom version.
The old variables have been replaced for a better support.

- [x] Code script review
- [ ] Sql review (double check the compatibility with the old variables)
- [x] Requires https://github.com/rathena/rathena/pull/4446 for achievement script

### CODE REVIEW

File: npc/re/quests/quests_16_1.txt [hunk 1]
```diff
@@ -13205,6 +13436,35 @@ prt_cas,98,248,5	script	Cat#ep16bgm	4_CAT,{
 	close;
 }
 
+prt_cas,90,121,5	script	Hiba Ajif#ep16	4_M_REDSWORD,{
```
Comment (Akaineko-astasi) [inline npc/re/quests/quests_16_1.txt]:
```suggestion
prt_cas,98,231,5	script	Hiba Ajif#ep16	4_M_REDSWORD,{
```

File: npc/re/quests/quests_16_1.txt [hunk 2]
```diff
@@ -13205,6 +13436,35 @@ prt_cas,98,248,5	script	Cat#ep16bgm	4_CAT,{
...
+	mes "[Instructor Igrid]";
+	mes "Brother, these shoes hurt my feet. May I go back to the guest room and sleep?";
+	next;
+	mes "[Hiba Ajif]";
+	mes "]If you can't avoid it, enjoy it...";
+	mes "I shall get drunk fast.";
+	close;
+}
+
+prt_cas,94,118,3	script	Avidal the Advisor#ep16	4_M_SAGE_C,{
```
Comment (Akaineko-astasi) [inline npc/re/quests/quests_16_1.txt]:
```suggestion
prt_cas,100,231,3	script	Avidal the Advisor#ep16	4_M_SAGE_C,{
```

File: npc/re/quests/quests_16_1.txt [hunk 3]
```diff
@@ -13205,6 +13436,35 @@ prt_cas,98,248,5	script	Cat#ep16bgm	4_CAT,{
...
+	mes "[Avidal the Advisor]";
+	mes "I have spent too much time in the tension between the two races. It just feels like I am dreaming to be in a happy and relaxing place like this. This must be what they call peace, right?";
+	close;
+}
+
+prt_cas,85,118,7	script	Instructor Igrid#ep16	4_M_CRU,{
```
Comment (Akaineko-astasi) [inline npc/re/quests/quests_16_1.txt]:
```suggestion
prt_cas,97,229,7	script	Instructor Igrid#ep16	4_M_CRU,{
```

**Comment by Akaineko-astasi** [comment]
`Levuiere Wigner#ep16_lib01`'s position is `prt_lib,88,83` (now they are stickied together with Maximillian Roegenburg)

File: npc/re/quests/quests_16_1.txt (hunk 1 above)
Comment (Atemo) [reply npc/re/quests/quests_16_1.txt:13450]: The npcs exist but they are commented in our version. Maybe they changed them with the server/version. Your reference is kro?

Comment (Akaineko-astasi) [reply npc/re/quests/quests_16_1.txt:13450]: Yep, kro sakray

File: sql-files/upgrades/upgrade_20200310.sql [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+DELETE FROM `char_reg_num` WHERE `key` = 'banquet_richard_quest';
+
+UPDATE `char_reg_num` SET `key` = 'ep16_wal' WHERE `key` = 'banquet_walther_quest' AND `value` < 2;
+UPDATE `char_reg_num` SET `key` = 'ep16_wal', `value` = `value` - 1 WHERE `key` = 'banquet_walther_quest';
+UPDATE `char_reg_num` SET `key` = 'ep16_lug' WHERE `key` = 'banquet_roegenburg_quest';
+UPDATE `char_reg_num` SET `key` = 'ep16_gaobs' WHERE `key` = 'banquet_geoborg_quest';
+
+UPDATE `char_reg_num` SET `key` = 'ep16_wig' WHERE `key` = 'banquet_wigner_quest' AND `value` < 5;
+UPDATE `char_reg_num` SET `key` = 'ep16_wig', `value` = `value` + 5 WHERE `key` = 'banquet_wigner_quest' AND `value` > 5;
+UPDATE `char_reg_num` SET `key` = 'ep16_wig', `value` = 10 FROM `char_reg_num` INNER JOIN `quest` ON `char_reg_num`.`char_id` = `quest`.`char_id`
```
Comment (Badarosk0) [inline sql-files/upgrades/upgrade_20200310.sql:17]: syntax error here?

**Comment by Lemongrass3110** [comment]
#4446 has been merged. So you should be able to make use of it now. :-)

**Pull request merged**